test/graphql_generator_test.dart

```diff
@@ -42,4 +42,57 @@ void main() {
       resultFile.delete();
     }
   });
+
+test('GraphQL generation', () async {
+    /// Read the test response from the file.
+    var graphQLGenerator = GraphQLGenerators();
+    var response = await graphQLGenerator.getSchema("https://turtle-api.herokuapp.com/graphiql", "", introspectionQuery:
+    '{"query" : "fragment FullType on __Type { kind name description fields(includeDeprecated: true) '
+              '{ name description args { ...InputValue } type { ...TypeRef } isDeprecated deprecationReason } '
+              'inputFields { ...InputValue } interfaces { ...TypeRef } enumValues(includeDeprecated: true) '
+              '{ name description isDeprecated deprecationReason } possibleTypes { ...TypeRef } } '
+              'fragment InputValue on __InputValue { name description type { ...TypeRef } defaultValue } '
+              'fragment TypeRef on __Type { kind name ofType { kind name ofType { kind name ofType '
+              '{ kind name ofType { kind name ofType { kind name ofType { kind name ofType { kind name } } } } } } } } '
+              'query IntrospectionQuery { __schema { queryType { name } mutationType { name } types { ...FullType } '
+              'directives { name description locations args { ...InputValue } } } } "}');
+    var typesFromResponse = getTypeList(
+        graphQLGenerator.getTypesFromResponse(response:response));
+    var mutationTypeName = graphQLGenerator.getMutationTypeNameFromResponse(
+        response: response);
+    var namespace = "T";
+
+    /// Generate the dart code and write it to a file.
+    var resultFile = await File('test/test_data/turtle_backend.schema.result.txt')
```

Comment:
Why are you writing it to a string? It seems like an unnecessary step to write the result from a file, pull it back out of the file, then delete the file when you are done. I would suggest storing the result in a string and skipping all fo this.

Reply:
There was an issue with the '\n' (newline) when the result string was compared to the string from file. 
When I save the result to the file and retrieve it as a string, the problem was resolved.